core/modules/node/node.pages.inc

```diff
@@ -704,12 +704,7 @@ function node_preview_banner_form_node_submit(array $form, array &$form_state) {
 function node_preview_banner_form_view_mode_submit(array $form, array &$form_state) {
   $node = $form_state['values']['node'];
   $node_tempstore_id = $form_state['values']['node_tempstore_id'];
-  if (isset($node->nid)) {
```

Comment:
Just want to confirm if you meant to commit this part. Doesn't seem related to the CSS.